src/Backend/Spec.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Command} from './Command';
+
+/**
+ * There are two type of Spec described on this file.
```

Comment:
I recommend
1. Here describes Spec concept. It would be great to support describption for the concept.
2. The line above DeviceSpec class describes DeviceSpec.
3. The line above BridgeSpec class describes BridgeSpec.